### PR TITLE
Fix typos in documentation and source comments (+ whitespace tweaks)

### PR DESCRIPTION
**Pull request by luzpaz**
Found via `codespell -q 3 -S ./docs/old,./old_nodes -L indicies,nd,nin,splitted,vertexes`

### CODE REVIEW

File: docs/nodes/curve/nearest_point.rst [hunk 1]
```diff
@@ -35,7 +35,7 @@ Inputs
 This node has the following inputs:
 
 * **Curve**. The curve to search the point on. This input is mandatory.
-* **Point**. The point to search the neares for. The default value is global origin ``(0, 0, 0)``.
+* **Point**. The point to search the nearets for. The default value is global origin ``(0, 0, 0)``.
```
Comment (zeffii) [inline docs/nodes/curve/nearest_point.rst:38]: nope. this is wrong.

File: utils/handling_nodes.py [hunk 2]
```diff
@@ -169,7 +169,7 @@ def get_name(self) -> str:
 
 
 class NodeProperties(NamedTuple):
-    bpy_props: tuple  # tuple is whet all bpy.props actually returns
+    bpy_props: tuple  # tuple is where all bpy.props actually returns
```
Comment (zeffii) [inline utils/handling_nodes.py:172]:
```diff
- bpy_props: tuple # tuple is whet all bpy.props actually returns
+ bpy_props: tuple # tuple is what all bpy.props actually return
```

File: utils/solid.py [hunk 3]
```diff
@@ -403,7 +403,7 @@ def __init__(self, solid, edge_mask=None, edge_map=None, face_mask=None, face_ma
 
 def transform_solid(matrix, solid):
     """
-    Utility funciton to apply mathutils.Matrix to a Solid object.
+    Utility functiton to apply mathutils.Matrix to a Solid object.
```
Comment (zeffii) [inline utils/solid.py:406]: this shlu.d be `function`.

**Review by zeffii — CHANGES_REQUESTED** [review]
i reviewed all 329 files, and found 3 issues.  
Nice work :)

**Comment by luzpaz** [comment]
Thanks and sorry about that! Requested fixes are in b0bbb3b

**Review by zeffii — APPROVED** [review]
small issues fixed. can commit.

**Comment by zeffii** [comment]
@luzpaz  thanks for the contribution!

**Pull request merged**